src/basic/path-util.c

```diff
@@ -1441,3 +1441,36 @@ int path_glob_can_match(const char *pattern, const char *prefix, char **ret) {
                 *ret = NULL;
         return false;
 }
+
+const char* default_path(void) {
+#if HAVE_SPLIT_BIN
+        static int split = -1;
+        int r;
+
+        /* Check whether /usr/sbin is separate and return the appropriate $PATH.
+         * On error assume we need split bin as configured… */
+
+        if (split < 0)
+                FOREACH_STRING(path, "/usr/sbin", "/usr/local/sbin") {
+                        _cleanup_free_ char *target = NULL;
+
+                        r = chase(path, /* root= */ NULL, /* flags= */ 0, &target, NULL);
+                        if (r == -ENOENT)
+                                continue;
+                        if (r >= 0 &&
+                            path_equal_or_inode_same(target,
```

Comment:
indentation looks wrong here?

Reply:
`p` is aligned to `r` above. I think it's all good.